packages/webdriver/src/utils.ts

```diff
@@ -273,6 +273,8 @@ export function getEnvironmentVars({ isW3C, isMobile, isIOS, isAndroid, isFirefo
         isSeleniumStandalone: { value: isSeleniumStandalone },
         isBidi: { value: isBidi },
         isChromium: { value: isChromium },
+        isNativeContext: { value: isNativeContext },
+        mobileContext: { value: mobileContext },
```

Comment:
We should define these environment flags in the `webdriverio` package instead. This will allow use to manage this value dynamically through the context manager, e.g.

```suggestion
        isNativeContext: { get: function () {
          const context = getContextManager(this)
          return context.isNativeContext
        } },
```

We should avoid having both flags being writeable as we don't want users mess around with it. I think we can achieve this easily with proposed approach. I suggest adding these flags [here](https://github.com/webdriverio/webdriverio/blob/9a14e93c358a0f527c8a462a98edd522bdf50fa5/packages/webdriverio/src/utils/index.ts#L52-L57).